sonar-db/src/main/java/org/sonar/db/ce/CeQueueDao.java

```diff
@@ -33,6 +33,8 @@
 
 public class CeQueueDao implements Dao {
 
+  private static final RowBounds ONE_ROW_LIMIT = new RowBounds(0, 1);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "ONE_ROW_LIMIT" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1068)